src/index.html

```diff
@@ -1,13 +1,355 @@
+              Audio sunglasses
+            </div>
+          </div>
+
+          <div class="grid grid--tablet-desktop">
+            <div class="grid__item grid__item--tablet--2-5 grid__item--desktop--5-8">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--tablet-desktop">
+            <div class="grid__item grid__item--tablet--1-3 grid__item--desktop--1-6">
+              <h2 class="page__section-title about__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+
+
+            <div class="grid__item grid__item--tablet--4-6 grid__item--desktop--7-11">
+              <ul class="about__advantages">
+                <li class="about__advantages-item">Free 2-day shipping and returns</li>
+                <li class="about__advantages-item">90-day risk-free trial</li>
+                <li class="about__advantages-item">World class customer service</li>
+                <li class="about__advantages-item">My Bose account management</li>
+              </ul>
+              <p class="about__text">A great product is more than what’s in the box. It’s a promise of premium
+                performance, world-class support, and
+                everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with
+                confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title contacts__title">Contact us</h2>
+
+          <div class="grid grid--tablet-desktop">
+            <div class="grid__item grid__item--tablet--1-3 grid__item--desktop--1-5">
+              <form
+                action="/"
+                method="get"
+                autocomplete="on"
+                id="contact-form"
+              >
+                <input
+                  class="form-field contacts__field"
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  required 
+                >
+                <input
+                  class="form-field contacts__field"
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  required
+                >
+                <textarea
+                  class="form-field form-field--text contacts__field contacts__field--last"
+                  name="message"
+                  placeholder="Message" 
+                  required
+                ></textarea>
+
+                <button class="button contacts__button">Send</button>
```

Comment:
I guess, that button should have type 'Submit'

Reply:
Thank you) It is important, I will fix it.